pandas/conftest.py

```diff
@@ -1,120 +1,25 @@
 """
-This file is very long and growing, but it was decided to not split it yet, as
-it's still manageable (2020-03-17, ~1.1k LoC). See gh-31989
+Just a conftest file for doctest stuff
```

Comment:
Sorry if I missed in the thread but why do we need a doctest conftest separate from the test confest? This doesn't work to move everything?